openslides_backend/action/actions/option/update.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from ....models.models import Option
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import ActionException
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ..vote.create import VoteCreate
+from ..vote.update import VoteUpdate
+
+
+@register_action("option.update")
```

Comment:
I guess this action should also be internal? I don't see why the client would or should ever set the values on an option directly

Reply:
If that's the case, you can probably fuse this with the `set_auto_fields` action from the other PR. We don't need 2 internal actions on the same fields.

Reply:
In the analog type we want to update options directly. So it is not internal.
The set_auto_fields does something different then this action.